src/include/kernel/scheduling/task.h

```diff
@@ -17,5 +18,13 @@ struct task_t {
     task_t* next;
 };
 
+extern size_t next_task_id;
+
 task_t* create_task(char *name, void (*_entry_point)(void *), void *args);
+void task_add_thread(thread_t* thread);
```

Comment:
I think this is leftover from a previous implementation: it has no definition anywhere